src/components/WalletConnect/NanoContract/NanoContractMethodArgs.js

```diff
@@ -76,7 +84,11 @@ export const NanoContractMethodArgs = ({ blueprintId, method, ncArgs }) => {
 
     const methodInfo = getMethodInfoFromBlueprint(blueprintInfo, method);
     if (methodInfo) {
-      return ncArgs.map((arg, idx) => [methodInfo.args[idx].name, arg]);
+      return ncArgs.map((arg, idx) => [
+        methodInfo.args[idx]?.name || t`Position ${idx}`,
+        arg,
+        methodInfo.args[idx]?.type || 'str',
```

Comment:
Why do you need the default values?

Reply:
The `ncArgs` can be called with more arguments that what is expected. If we fall in this scenario, the fallback value covers it.

Reply:
Fallback for extra arguments removed.